src/main/java/com/theelixrlabs/healthcare/service/PatientService.java

```diff
@@ -22,56 +25,49 @@ public class PatientService {
 
     private final MessageSource messageSource;
```

Comment:
Make sure to remove message source as it is only getting used to send as argument

Reply:
Resolved.